drivers/pwm/pwm_renesas_ra.c

```diff
@@ -39,6 +40,8 @@ struct pwm_renesas_ra_data {
 	gpt_instance_ctrl_t fsp_ctrl;
 	timer_cfg_t fsp_cfg;
 	gpt_extended_cfg_t extend_cfg;
+	struct renesas_elc_dt_spec start_renesas_elc;
+	struct renesas_elc_dt_spec stop_renesas_elc;
```

Comment:
It should be wrapped by RENESAS_RA_ELC
```suggestion
#ifdef CONFIG_RENESAS_RA_ELC
	struct renesas_elc_dt_spec start_renesas_elc;
	struct renesas_elc_dt_spec stop_renesas_elc;
#endif /* CONFIG_RENESAS_RA_ELC */
```